CODING_GUIDELINES.md

```diff
@@ -211,6 +211,61 @@ a += "b"
   + "d";
 ```
 
+* Initializers
+
+We allow brace enclosed initializers only for aggregates and arrays/containers.
+Brace enclosed initializer MUST be used with equality sign if it follow the variable declaration.
+Brace enclosed initializer MUST be additionally enclosed in parentheses if it used in constructor initialization list.
+Some valid use cases:
```

Comment:
I think here you wanted newlines on every sentence, correct?
The rendered text doesn't have them.
IIRC there are 2 ways. Leave an empty space at the end or put `<br/>` at the end.